web/src/main/java/org/cbioportal/weblegacy/GeneController.java

```diff
@@ -49,8 +52,13 @@
     @Autowired
     private GeneService geneService;
 
+    @ApiOperation(value = "Get all information about genes from hugo symbols including entrezGeneId, hugoGeneSymbol, cytoband, length and chromosome",
+            nickname = "getGeneListByHugoSymbols",
+            notes = "")
+    @Transactional
     @RequestMapping(method = {RequestMethod.GET, RequestMethod.POST}, value = "/gene/fetch-by-hugo")
-    public List<Gene> getGeneListByHugoSymbols(@RequestParam(required = true) List<String> hugo_gene_symbols) {
+    public List<Gene> getGeneListByHugoSymbols(@ApiParam(value = "a list of gene hugo symbols that you want to query on")
```

Comment:
An example case .. or directions to the /api/gene endpoint would help the user understand better